cmd/antrea-agent/agent.go

```diff
@@ -188,7 +188,9 @@ func run(o *Options) error {
 		IPsecConfig: config.IPsecConfig{
 			AuthenticationMode: ipsecAuthenticationMode,
 		},
+		EnableMulticlusterGW: enableMulticlusterGW,
 	}
+	networkConfig.MTUDeduction = networkConfig.CalculateMTUDeduction()
```

Comment:
Probably add a `config.NewNetworkConfig()` func, and initialize networkConfig.MTUDeduction there.

Reply:
Moved CalculateMTUDeduction to agent initialization, so didn't add a new function for this.